docs/spec_draft.md

```diff
@@ -38,8 +41,8 @@ immutable n-dimensional arrays and are referred to in the document as
   `si64` and are non-negative (dimension sizes equal to zero are allowed,
   and their meaning is described below). Ranks equal to zero are also allowed,
   and their meaning is also described below.
-  * **Element type** `E` is any one of the supported element types mentioned
-  above.
+  * **Element type** `E` is any one of the supported element types, except
+  the token type, mentioned above.
```

Comment:
Why is token type defined in the section dedicated to element types, when it's not an element type? Let's define it e.g. right above the "Function types" section? Then this change can be reverted.